app/src/main/java/com/example/mobilelibrary/ui/dashboard/ImageTextAnalyzer.java

```diff
@@ -102,4 +141,116 @@ public void analyze(ImageProxy image, int rotationDegrees) {
+                                    public void onFailure(@NonNull Exception e) {
+                                        // Task failed with an exception
+                                        // ...
+
+
+                                    }
+                                });*/
+        Tasks.await(result);
+        FirebaseVisionText extractedResult = result.getResult();
+        String text= extractedResult.getText();
+        splitText(extractedResult);
+
+
+
+    }
+
+    public void splitText(FirebaseVisionText fb) {
+        List <String> lines = null;
+        for (FirebaseVisionText.TextBlock block : fb.getTextBlocks()) {
+            //TODO: Catalin
+            String text= block.getText();
+            String translation= translate(text);
+            }
+        }
+
+    public String translate(String text)
+    {
+        // Create an English-German translator:
+        TranslatorOptions options =
+                new TranslatorOptions.Builder()
+                        .setSourceLanguage(TranslateLanguage.ENGLISH)
+                        .setTargetLanguage(TranslateLanguage.ROMANIAN)
+                        .build();
+        final Translator englishRomanianTranslator =
+                Translation.getClient(options);
+
+        DownloadConditions conditions = new DownloadConditions.Builder()
+                .requireWifi()
+                .build();
+        englishRomanianTranslator.downloadModelIfNeeded(conditions)
+                .addOnSuccessListener(
+                        new OnSuccessListener<Void>() {
+                            @Override
+                            public void onSuccess(Void v) {
+                                // Model downloaded successfully. Okay to start translating.
+                                // (Set a flag, unhide the translation UI, etc.)
+                            }
+                        })
+                .addOnFailureListener(
+                        new OnFailureListener() {
+                            @Override
+                            public void onFailure(@NonNull Exception e) {
+                                // Model couldn’t be downloaded or other internal error.
+                                // ...
+                            }
+                        });
+
+        englishRomanianTranslator.translate(text)
+            .addOnSuccessListener(
+                    new OnSuccessListener<String>() {
+                        @Override
+                        public void onSuccess(@NonNull String translatedText) {
+
+
+
+                        }
+                    })
+            .addOnFailureListener(
+                    new OnFailureListener() {
+                        @Override
+                        public void onFailure(@NonNull Exception e) {
+
+
+
+                        }
+                    });
+
+        String translated= englishRomanianTranslator.toString();
+        return translated;
+    }
+    
+
```

Comment:
this has already been addresed in a different pr. please remove and commit again